client/js/main.js

```diff
@@ -101,20 +180,32 @@ function getImage(imgData, canvas) {
 		canvas.height = this.naturalHeight;
 		context.drawImage(this, 0, 0);
 
-		drawSquares(imgData.faces, context);
+		drawSquares(imgData.faces, context, xShrunkBy, yShrunkBy);
 	}
 
 }
 
-function drawSquares(faces, context) {
+function drawSquares(faces, context, xShrunkBy=1.0, yShrunkBy=1.0) {
 	for(var i = 0; i < faces.length; ++i) {
-		var pos = faces[i].coordinates;
-		context.strokeStyle = (faces[i].gender === "man")?"blue":"red";
+		let face = faces[i];
```

Comment:
This is client-side, the use of `var` is recommended over `let`/`const`, as we're not compiling this.